crypto/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "crypto"
+version = "0.1.0"
+edition = "2021"
```

Comment:
I'd also add other properties which we usually add to crates (description, license etc.)